browser-extension/src/utils/api-client.ts

```diff
@@ -8,11 +8,14 @@ import {
   Producer,
   Wallet,
 } from "../types";
-import persistentStorage from "./persistentStorage";
+import persistentStorage from "./persistent-storage";
+import applyCaseMiddleware from "axios-case-converter";
 
-const api = axios.create({
-  baseURL: "http://localhost:8000",
-});
+const api = applyCaseMiddleware(
+  axios.create({
+    baseURL: "http://localhost:8000",
+  })
+);
```

Comment:
This will convert between snake case (used by api) and camel case (used by frontend) JSON keys